tsconfig.json

```diff
@@ -61,6 +61,7 @@
     }
   },
   "include": [
-    "src"
+    "src",
+    "example"
```

Comment:
This might be removed.

Reply:
Amazingly, yes. We definitely had problems yesterday that were solved with this, but apparently not any more. Fixed with 1fe2c12